care/emr/api/viewsets/service_request.py

```
@@ -0,0 +1,294 @@
+    questionnaire_subject_type = SubjectType.patient.value
+    resource_type = TagResource.service_request
+    ordering_fields = ["created_date", "modified_date"]
+
+    def get_facility_obj(self):
+        return get_object_or_404(
+            Facility, external_id=self.kwargs["facility_external_id"]
+        )
+
+    def convert_external_id_to_internal_id(self, instance):
+        ids = []
+        for location in instance._locations:
```

Ruff (0.11.9)

Private member accessed: `_locations`

(SLF001)